### PR TITLE
Make python-fastpbkdf2 build with .tar.gz file.

### PR DESCRIPTION
**Pull request by Ayrx**

### CODE REVIEW

File: src/build_fastpbkdf2.py [hunk 1]
```diff
@@ -8,9 +8,19 @@
...
 
+import tarfile
+
+import tempfile
+
 from cffi import FFI
 
-DIR = os.path.join(os.path.dirname(__file__), "c")
+
+TAR_DIR = os.path.join(os.path.dirname(__file__), "fastpbkdf2.tar.gz")
```
Comment (reaperhulk) [inline src/build_fastpbkdf2.py]: This isn't the dir, it's the full path to the tarball

**Comment by reaperhulk** [comment]
With the caveat that I'm largely unfamiliar with the stdlib `tarfile` this LGTM.

**Comment by Ayrx** [comment]
@reaperhulk Cool. Should be good if I open PRs for bcrypt and pynacl with this same approach?

**Comment by reaperhulk** [comment]
Let's do bcrypt first as that's a nice place to experiment. We can push a new release and make sure everything's reliable :)

**Pull request merged**